setup.py

```diff
@@ -552,6 +552,48 @@ def load(filename):
             with open('compile_commands.json', 'w') as f:
                 f.write(new_contents)
 
+class concat_license_files():
+    """Merge LICENSE.txt and LICENSES_BUNDLED.txt as a context manager
+
+    Done this way to keep LICENSE.txt in repo as exact BSD 3-clause.
+    This makes GitHub web gui state correctly how the repo is licensed.
```

Comment:
These two sentences aren't true, I'd fix that. The main `LICENSE.txt` file needs to be exactly BSD-3 for it to be recognized, which I don't think you can do - there's too much copyright info in it for example.

This should say something like:
_`LICENSE.txt` is the main PyTorch license, `LICENSES_BUNDLED.txt` is auto-generated from all the licenses found in third_party/._